blog/2015/04/how-to-contribute-with-template.html

```diff
@@ -0,0 +1,112 @@
+---
+layout: blog
+root: ../../..
+author: Raniere Silva
+title: "How to send pull request to lesson template"
+date: 2015-04-04
+time: "22:00:00"
+category: ["How to"]
+---
+<!-- start excerpt -->
+<p><a href="../../2014/09/splitting-the-repo.html">At the end of last year</a> we split out lessons to have one Git repository per topic that we teach. To ensure that all lessons have the same look we create <a href="https://github.com/swcarpentry/lesson-template/">one template for our lessons</a>. During the first quarter of this year we found that have two living branches at the same remote Git repository is confusing for many people and to address this we now have the core files of our template <a href="https://github.com/swcarpentry/lesson-template/pull/219">in one repository</a> and the example of how it looks at <a href="https://github.com/swcarpentry/lesson-example">another repository</a>. Because create a pull request to our lesson template could be not clear for someone starting with Git I wrote this how-to.</p>
+<!-- end excerpt -->
```

Comment:
I agree with @karinlag - we need to make really, really sure people understand that they don't have to do all of this to add to a lesson.